netscaler/client.go

```diff
@@ -44,6 +45,8 @@ type NitroClient struct {
 	password  string
 	proxiedNs string
 	client    *http.Client
+	sessionid string
```

Comment:
`sessionid` is is not initialized in the constructor. I realize that the default value is an empty string..